docs/examples/plotting_functions/mesh.md

```diff
@@ -2,6 +2,33 @@
 
 {{doc mesh}}
 
+## Attributes
+
+### Generic
+
+- `visible::Bool = true` sets whether the plot will be rendered or not.
```

Comment:
I think ideally, we could put these in doc strings in code somewhere, so it's easier to keep it synchronized with the code

Reply:
I think so too, but I'd say we should put in a bit more effort than just a raw docstring. Lots of recipes inherit attributes and it would make sense if they inherit docstrings as well. (It would also be nice to have some filtering, e.g. filtering attributes applicable to `lines` to more easily pass attributes. But that's kinda beside the point.)

I didn't want to bother with this here though. I just wanted to make a quick pr documenting things since I keep seeing people ask about it.

Reply:
Fair enough, lets leave it like that then ;)